src/store/index.js

```diff
@@ -3,9 +3,37 @@ import Vuex from "vuex";
 
 Vue.use(Vuex);
 
-export default new Vuex.Store({
-  state: {},
-  mutations: {},
-  actions: {},
+const store = new Vuex.Store({
+  state: {
+    user: ""
+  },
+  mutations: {
+    SET_USER(state, user) {
+      state.user = user;
+    },
+    LOGOUT(state) {
+      state.user = "";
+    },
+    CYPRESS_SET_USER(state, user) {
+      state.user = user;
+    }
```

Comment:
Why a separate `CYPRESS_SET_USER` mutation? Doesn't `SET_USER` do the same thing?